modules/ethereum/src/lib.rs

```diff
@@ -100,7 +143,7 @@ pub struct HeaderToImport<Submitter> {
 	/// Total chain difficulty at the header.
 	pub total_difficulty: U256,
 	/// Validators set enacted change, if happened at the header.
-	pub enacted_change: Option<Vec<Address>>,
+	pub enacted_change: Option<(Option<H256>, Vec<Address>)>,
```

Comment:
So is this tuple `(Header at which change was enacted, authority set enacted by this change)`?

Reply:
No :) The change is enacted by this (importing) header. So we need pointer to the block where this change has been scheduled. I've updated comment. Could replace with struct if you feel it would be better.

Reply:
I think typedef-ing it would be cleaner